### DISCUSSION TITLE
Send a PLI on an interval, This is a temporary fix until we implement incoming RTCP events

### DISCUSSION THREAD

**Discussion by katepangLiu** [pion/webrtc#1917] [Q&A]
**Is that means pion doesn't support incoming RTCP event handling,  or the examples like reflect doesn't support yet? Thanks.**

https://github.com/pion/webrtc/blob/5047c6b01014ccbfb38907b6dc08294cf9ec9ebf/examples/reflect/main.go#L101
https://github.com/pion/webrtc/blob/5047c6b01014ccbfb38907b6dc08294cf9ec9ebf/examples/reflect/main.go#L102

**Comment by Sean-Der** [comment]
Hi @katepangLiu 

Pion does support responding to RTCP events. I would look at peerconnection_media_tests.go to see how it works.

I will add an example that shows processing/responding to RTCP
